### ISSUE TITLE
[Feature Request] Add support for dark/tinted icons

### ISSUE THREAD

**Issue by acandoo** [Ranchero-Software/NetNewsWire#4364]
iOS 18 introduces dark and tinted icons. Adding support for this would make the app look more in line with others.

**Comment by plgagne** [comment]
Yeah with the current icon being a simple shape with a single background, it would be trivial to implement if we had access to the vector files to make the icon. Right now, all I can see in the project are the final versions for distribution. Where are the original vector files?

**Comment by djpowers** [comment]
I found this searching to see if a [dark mode version of the icon for iOS](https://developer.apple.com/design/human-interface-guidelines/app-icons#iOS-iPadOS) had been suggested. This would help NetNewsWire blend in better for users with dark icons enabled. Additionally I think the satellite iconography would look really good over a dark background. 🛰️🌑

**Comment by brentsimmons** [comment]
We will do all this, but we can’t predict as to when that will be 👍

**Comment by ivanreese** [comment]
@brentsimmons I'd be happy to take a swing at adding this, if you'd accept a PR.

**Comment by brentsimmons** [comment]
Thanks for the offer! But our app icon designer is on the case 👍
